### ISSUE TITLE
Log in using credentials VRTMAX ends in "HTTP Error 500: Internal Server Error" after retrieving and downloading token

### ISSUE THREAD

**Issue by videobeeldhouwer** [yt-dlp/yt-dlp#7574]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Belgium

### Provide a description that is worded well enough to be understood

Logging in  using credentials gets followed by retrieving and downloading (VRT) token but ends in error (HTTP Error 500: Internal Server Error).

using yt-dlp -f b -u "user" -p "pass" https://www.vrt.be/vrtnu/a-z/x/2023/y/ , gives

"Retrieving vrtnutoken" followed by "Downloading player token", ending in 

"Unable to download JSON metadata: HTTP Error 500: Internal Server Error" (...please report this issue...)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU -f b -u "user" -p "pass" https://www.vrt.be/vrtnu/a-z/terzake/2023/terzake-d20230712/
[debug] Command-line config: ['-vU', '-f', 'b', '-u', 'PRIVATE', '-p', 'PRIVATE', 'https://www.vrt.be/vrtnu/a-z/terzake/2023/terzake-d20230712/']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.11.4 (CPython x86_64 64bit) - macOS-13.4.1-x86_64-i386-64bit (OpenSSL 3.1.1 30 May 2023)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[VrtNU] Logging in
[VrtNU] Requesting XSRF Token
[VrtNU] Performing login
[VrtNU] Extracting URL: https://www.vrt.be/vrtnu/a-z/terzake/2023/terzake-d20230712/
[VrtNU] terzake-d20230712: Downloading asset JSON
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Retrieving vrtnutoken
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Downloading player token
ERROR: [VrtNU] terzake-d20230712: Unable to download JSON metadata: HTTP Error 500: Internal Server Error (caused by <HTTPError 500: 'Internal Server Error'>); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 710, in extract
    ie_result = self._real_extract(url)
                ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/vrt.py", line 289, in _real_extract
    video_info = self._call_api(video_id, 'vrtnu-web@PROD', vrtnutoken)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/vrt.py", line 90, in _call_api
    player_token = self._download_json(
                   ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 1064, in download_content
    res = getattr(self, download_handle.__name__)(url_or_request, video_id, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 1028, in download_handle
    res = self._download_webpage_handle(
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 898, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query, expected_status=expected_status)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 855, in _request_webpage
    raise ExtractorError(errmsg, cause=err)

  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/extractor/common.py", line 837, in _request_webpage
    return self._downloader.urlopen(self._create_request(url_or_request, data, headers, query))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/yt-dlp/2023.7.6/libexec/lib/python3.11/site-packages/yt_dlp/YoutubeDL.py", line 3863, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 525, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 634, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 563, in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 496, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.4_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 500: Internal Server Error
```

**Comment by bashonly** [comment]
I think this was fixed by 325191d0c9bf3fe257b8a7c2eb95080f44f6ddfc, but I don't have an account to test

Try installing from master branch with pip:
```
python3 -m pip install -U pip setuptools wheel
python3 -m pip install --force-reinstall https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz
```

**Comment by videobeeldhouwer** [comment]
I've figured out that credentials are not necessary to download. Just need to find the " .ism" link in the page source and add /manifest .
This works perfectly for non DRM content. 
ex. 

yt-dlp https://ondemand.vrtcdn.be/content/vod/vid-210e1c2b-8605-434c-a94b-f28112274b3f-CDN_2_HD2022/vid-210e1c2b-8605-434c-a94b-f28112274b3f-CDN_2_HD2022_nodrm_bebc2f56-6cfc-45b6-8983-755a8e56db6a.ism/manifest

**Comment by bashonly** [comment]
@videobeeldhouwer  Did you try downloading after installing from master branch? Or could you try it? I am pretty sure this issue is solved and having confirmation from an account-holder would be nice

**Comment by videobeeldhouwer** [comment]
@bashonly I can confirm that it works perfectly after installing master branch with pip and using credentials.

( My previous install was up to date but installed via Homebrew)

Grateful, thank you.

Verbose output:

yt-dlp -vU -f b -u "--" -p "--" https://www.vrt.be/vrtnu/a-z/terzake/2023/terzake-d20230712/
[debug] Command-line config: ['-vU', '-f', 'b', '-u', 'PRIVATE', '-p', 'PRIVATE', 'https://www.vrt.be/vrtnu/a-z/terzake/2023/terzake-d20230712/']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481]
[debug] Lazy loading extractors is disabled
[debug] Python 3.11.4 (CPython x86_64 64bit) - macOS-13.4.1-x86_64-i386-64bit (OpenSSL 3.1.1 30 May 2023)
[debug] exe versions: ffmpeg 6.0 (setts), ffprobe 6.0, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[VrtNU] Logging in
[VrtNU] Requesting XSRF Token
[VrtNU] Performing login
[VrtNU] Extracting URL: https://www.vrt.be/vrtnu/a-z/terzake/2023/terzake-d20230712/
[VrtNU] terzake-d20230712: Downloading asset JSON
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Retrieving vrtnutoken
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Downloading player token
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Downloading API JSON
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Downloading m3u8 information
[VrtNU] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Downloading MPD manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[info] pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3: Downloading 1 format(s): HLS-2426
[debug] Invoking hlsnative downloader on "https://ondemand.vrtcdn.be/content/vod/vid-356faeb2-3d80-41b2-b872-7f65c8785ab3-CDN_6_HD2022/vid-356faeb2-3d80-41b2-b872-7f65c8785ab3-CDN_6_HD2022_nodrm_17f6573f-1042-466b-a47e-e626ae980f99.ism/vid-356faeb2-3d80-41b2-b872-7f65c8785ab3-CDN_6_HD2022_nodrm_17f6573f-1042-466b-a47e-e626ae980f99-audio=96000-video=2191693.m3u8"
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 358
[download] Destination: Terzake 20230712 [pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3].mp4
[download] 100% of  607.10MiB in 00:01:21 at 7.49MiB/s
[debug] ffprobe command line: ffprobe -hide_banner -show_format -show_streams -print_format json 'file:Terzake 20230712 [pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3].mp4'
[debug] ffmpeg command line: ffprobe -show_streams 'file:Terzake 20230712 [pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3].mp4'
[FixupM3u8] Fixing MPEG-TS in MP4 container of "Terzake 20230712 [pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:Terzake 20230712 [pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3].mp4' -map 0 -dn -ignore_unknown -c copy -f mp4 -bsf:a aac_adtstoasc -movflags +faststart 'file:Terzake 20230712 [pbs-pub-25ecfb8b-077c-4a14-9c95-ccb840310e46$vid-356faeb2-3d80-41b2-b872-7f65c8785ab3].temp.mp4'
